portability-api/src/main/java/org/dataportabilityproject/webapp/CryptoHelper.java

```diff
@@ -53,15 +52,15 @@
    * Serialize and encrypt the given {@code authData} with the session key.
```

Comment:
worker instance public key, not session key?

Reply:
Documented the different methods with SecretKey and PublicKey